agent/connection.go

```diff
@@ -75,7 +76,7 @@ func (a *Agent) sender(stream eventstreamapi.EventStream_SubscribeClient) error
 	}
 	logCtx.Tracef("Grabbed an item")
 	if item == nil {
-		// FIXME: Is this really the right thing to do?
+		// TODO: Is this really the right thing to do?
```

Comment:
Changed `FIXME` to `TODO`. This seemed to be the only instance of `FIXME` in the code. Happy to revert this part of the PR if `FIXME` was intentional.